### PR TITLE
[blog] How we used OpenTelemetry to fix a bug in OpenTelemetry

### PR DESCRIPTION
**Pull request by svrnm**
A blog post I really enjoyed writing with @sanketmehta28 & @kpratyus :-)

Preview: https://deploy-preview-1721--opentelemetry.netlify.app/blog/2022/debug-otel-with-otel/

### CODE REVIEW

File: content/en/blog/2022/debug-otel-with-otel/index.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+![A screenshot of the Jaeger UI showing that http.request.header.traceparent has multiple entries.](jaeger-with-request-headers.png)
+
+There it is! The trace headers (`baggage`, `traceparent`, `tracestate`) are send
+as multiple header fields.
+
+Doing some research on that topic, we found out, that this is coved by
+[RFC7230][] and is [allowed and defined][] for `tracestate`, but the [definition
+for `traceparent`][] does not provide details on that corner-case.
+
+Although we didn't read into the source code of the OTel Java SDK & OTel Python
```
Comment (cartermp) [inline content/en/blog/2022/debug-otel-with-otel/index.md:217]: Should this manifest in issues for otel-py and otel-js?

**Review by cartermp — APPROVED** [review]
Really enjoyed the post!

File: content/en/blog/2022/debug-otel-with-otel/index.md (hunk 1 above)
Comment (svrnm) [reply content/en/blog/2022/debug-otel-with-otel/index.md:217]: I don't know what the expected behavior is, since the w3c spec does not define that.

**Review by austinlparker — APPROVED** [review]

**Comment by svrnm** [comment]
We're live: https://opentelemetry.io/blog/2022/debug-otel-with-otel/ 🎉

**Pull request merged**